### PR TITLE
Make dots work

### PR DESCRIPTION
**Pull request by carlydotwebsite**
Makes the dots work on the character sheet.

https://tree.taiga.io/project/stephmarx-ask-again-later-character-system/task/82?kanban-status=856838

### CODE REVIEW

File: app/controllers/storytellers/true_selves_controller.rb [hunk 1]
```diff
@@ -27,17 +27,17 @@ def update
...
-        redirect_to edit_data_entry_true_self_path(@true_self)
+        redirect_to edit_storytellers_true_self_path(@true_self)
       end
     end
 
     def destroy
       @true_self = TrueSelf.find(params[:id])
       @true_self.delete
-      redirec_to data_entry_true_selves_path
+      redirec_to storytellers_true_selves_path
```
Comment (korey-enright) [inline app/controllers/storytellers/true_selves_controller.rb]: should be redirect_to :)

**Review by korey-enright — CHANGES_REQUESTED** [review]

File: app/controllers/storytellers/true_selves_controller.rb (hunk 1 above)
Comment (carlydotwebsite) [reply app/controllers/storytellers/true_selves_controller.rb]: fixed!

**Comment by sublimemarch** [comment]
👍

**Review by korey-enright — APPROVED** [review]
:+1:

**Pull request merged**